analytics/config/config.go

```diff
@@ -43,15 +44,28 @@ func NewPBSAnalytics(analytics *config.Analytics) analytics.PBSAnalyticsModule {
 // Collection of all the correctly configured analytics modules - implements the PBSAnalyticsModule interface
 type enabledAnalytics []analytics.PBSAnalyticsModule
 
+// Returns the name of the analytics module
+func (ea enabledAnalytics) GetName() string {
+	return "enabledAnalytics"
```

Comment:
This is confusing. It doesn't make sense for `enabledAnalytics` to have a name since it's just aggregating actual modules. IMHO this is either pointing to either `GetName()` not being appropriate for the `PBSAnalyticsModule` type or `enabledAnalytics` shouldn't be a `PBSAnalyticsModule`.

I'm leaning towards the later. Could be a new object within the `analytics` package simply named `Engine` or some such. `analytics.Engine` / `analytics.Runner` / `analytics.Aggregator`etc.. looks nice to me.

For each request, we can create a the default one, but once we have account config we can replace it locally with a new object given a reference to the account analytics config.